src/styles/styles.js

```diff
@@ -2166,6 +2166,10 @@ const styles = {
         ...whiteSpace.noWrap,
     },
 
+    defaultDeletePopover: {
+        width: variables.sideBarWidth - 68,
```

Comment:
What is `68` representing? Can we create a variable or CONST for it?

Reply:
It's basically just how wide I wanted it looks in the mockups. Nothing specific, so I'm not sure what the const would be

Reply:
Whoa haha. Ok cool. it can be a NAB then but ya using `sidebarPopover` sounds good.

Reply:
Lol, I actually took that value from _this_  PR when I was trying to copy `ConfirmPopover`